contrib/utilities/automatic_launch/launch_lethe_locally.py

```diff
@@ -0,0 +1,20 @@
+"""
+Summary: Script to launch all the Lethe simulations, one at a time locally.
```

Comment:
I think it is important to clarify (also in the documentation) that the applications need to be part of your PATH (a warning or smth). On my machine, the launch script didn't work because of this.